From dc03f0fbb400bad5c4748eb969f4a85c68e3bc0c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Duje=20Mihanovi=C4=87?= <duje.mihanovic@skole.hr>
Date: Wed, 26 Jul 2023 21:14:34 +0200
Subject: [PATCH] DONOTMERGE: Add pxa,rev-id to board dts

This won't be accepted upstream, but it's needed to boot with just
S-Boot, so add this here.
---
 arch/arm64/boot/dts/marvell/pxa1908-samsung-coreprimevelte.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/marvell/pxa1908-samsung-coreprimevelte.dts b/arch/arm64/boot/dts/marvell/pxa1908-samsung-coreprimevelte.dts
index e3c863d517a8..04fb0566ed2b 100644
--- a/arch/arm64/boot/dts/marvell/pxa1908-samsung-coreprimevelte.dts
+++ b/arch/arm64/boot/dts/marvell/pxa1908-samsung-coreprimevelte.dts
@@ -4,6 +4,7 @@
 #include <dt-bindings/input/linux-event-codes.h>
 
 / {
+	pxa,rev-id = <3928 2>;
 	model = "Samsung Galaxy Core Prime VE LTE";
 	compatible = "samsung,coreprimevelte", "marvell,pxa1908";
 
-- 
2.42.0

